tests/examples/pacer/nef/s3/alsd_1.txt

```diff
@@ -0,0 +1,848 @@
+        d=1e100.net; s=20161025;
+        h=x-gm-message-state:from:mime-version:date:subject:message-id
+         :references:to;
+        bh=fIkeU4ZtwX18JOU+2PsJ5nL1whVrANOUjv0xSWPpk7Y=;
+        b=f2Mdltjrugpwe8VwE1hCa9txNTZ4iwApc5UtJJ531Zg9sndwdD91f/lP2dzecQE0uf
+         ff/VpU0JHLKNw5yZrzgu2KYHm1q1TteojlPxOe0gKqM9zOgmw8E1OLZ3LLn3JRQ/r3gJ
+         dZvjrttgA1NcE8pB9MRy2Q2fUs+OvprzsBZfnckxQrJiiXekbd6UFNV6LKjKZZN8KFt/
+         qRIp5hMSzE6X0UJmHSepBMnmcM2mPCgWWrLEMh3gDymoX6Ee4KtIhCb82q+NaM47FIEJ
+         Byz+dP1HulrAp08ZHCJ4NnGgcTZNQHbTHRRf+2NiEYa79o08pFQdKwUhwGCG12F5GbH5
+         FfZg==
+X-Gm-Message-State: AOAM531PYECe6a+ThtHdDEyXrIxzDIg2TNjACRpHCFrl4Q2BgQ6Hm+z+
+	Jru1/nsvTYMh79z1GU6M8lx7Fkr4Jk4jtdRfSU0AyqVoHIEBPQ==
+X-Google-Smtp-Source: ABdhPJwzStZy31Q6HfMTGvCVn1JMb/dhBF1TLsEtLMMGtO16ipm+ZLhlL9D2VQ0wJXqlJU0xdQfZ143RM1h5
+X-Received: by 2002:a17:902:d213:b029:127:9520:7649 with SMTP id t19-20020a170902d213b029012795207649mr16841337ply.10.1624897291795;
+        Mon, 28 Jun 2021 09:21:31 -0700 (PDT)
+Return-Path: <driscoll@everettlawfirm.com>
+Received: from us3.smtp.exclaimer.net (us3.smtp.exclaimer.net. [23.100.16.236])
+        by smtp-relay.gmail.com with ESMTPS id 1sm7662704pjf.2.2021.06.28.09.21.31
+        for <pacer@everettlawfirm.business>
+        (version=TLS1_2 cipher=ECDHE-ECDSA-AES128-GCM-SHA256 bits=128/128);
+        Mon, 28 Jun 2021 09:21:31 -0700 (PDT)
+X-Relaying-Domain: everettlawfirm.com
+Received: from mail-io1-f72.google.com (209.85.166.72) by
+	 us3.smtp.exclaimer.net (23.100.16.236) with Exclaimer Signature Manager
+	 ESMTP Proxy us3.smtp.exclaimer.net (tlsversion=TLS12,
+	 tlscipher=TLS_ECDHE_WITH_AES256_SHA1); Mon, 28 Jun 2021 16:21:31 +0000
+X-ExclaimerHostedSignatures-MessageProcessed: true
+X-ExclaimerProxyLatency: 9380383
+X-ExclaimerImprintLatency: 2800519
+X-ExclaimerImprintAction: 642559f1694b4c2b89bf508ea2d6449c
+Content-Type: multipart/related;
+	boundary="----_=_NextPart_d488229e-24b7-4f49-a62d-de749ba348c9"
+Received: by mail-io1-f72.google.com with SMTP id x21-20020a5d99150000b02904e00bb129f0so13917910iol.18
+        for <pacer@everettlawfirm.business>; Mon, 28 Jun 2021 09:21:30 -0700 (PDT)
+X-Received: by 2002:a6b:3c01:: with SMTP id k1mr229867iob.24.1624897290247;
+        Mon, 28 Jun 2021 09:21:30 -0700 (PDT)
+X-Received: by 2002:a6b:3c01:: with SMTP id k1mr229855iob.24.1624897290006;
+        Mon, 28 Jun 2021 09:21:30 -0700 (PDT)
+Return-Path: <driscoll@everettlawfirm.com>
+Received: from smtpclient.apple ([2600:380:c44c:6a1:513a:76f6:1034:2db])
+        by smtp.gmail.com with ESMTPSA id w11sm8581872ilc.8.2021.06.28.09.21.29
+        (version=TLS1_3 cipher=TLS_AES_128_GCM_SHA256 bits=128/128);
+        Mon, 28 Jun 2021 09:21:29 -0700 (PDT)
+From: Driscoll Wiley <driscoll@everettlawfirm.com>
+Mime-Version: 1.0
+Date: Mon, 28 Jun 2021 11:21:28 -0500
+Subject: Fwd: Activity in Case 1:21-cv-00057-WS-N Troxel v. Gunite Pros, LLC et al Document Noted

```

Comment:
This is a forwarded message, right? It has a lot of parts to it that seem out of scope. It's probably harder, but shouldn't we try to get the original messages instead of the forwarded ones?

Reply:
I have a lot of originals now that we are in prod for Sanford. 

Let me bring some of those over, but maybe we keep these and maintain our resilience, since we have already done the work. If the forwards end up causing too much of an issue, I can get rid of them.

Reply:
Brought in 10 more direct from Sanford.